### ISSUE TITLE
windows phone support

### ISSUE THREAD

**Issue by Tset-Noitamotua** [beeware/briefcase#24]
Are there plans for BeeWare to support Windows Phone 10?

**Comment by freakboy3742** [comment]
Broadly, yes - the plan is to support any device that has a user interface. However, the priority on Windows Phone is quite far down my personal list. 

That said, if someone who *does* have an interest in that (or any other) platform were to contribute a backend, I would merge it in (assuming it followed API/code standards, etc).

**Comment by Tset-Noitamotua** [comment]
I understand - sure market share of Windows Phone is not really worth mentioning. I like the look and feel of it's UI much more than the one of iOS and Android though. I hope that more people realize that soon, too.

This will be an epic post then people look at it in a few years :-D :-D :-D

> "When I look into the future I see a day there Windows Phone's market share will exceed the one of iOS and Android together! Android OS always sucked but they always had cheap phones / tablets for the masses. iOs was great prio to version 7 but now sucks and it's hardware is not affordable for the masses. The great UI of Windows Phone + affordable hardware will let Microsoft conquer market share attracting more developers and thus lead to an even better user experience for Windows Phone users!" (me)

:-)

**Comment by drafter250** [comment]
Hi I just found out about beeware from the "Talk Python to me" podcast. I have been looking for a good python windows packager! I started doing a little research and from what i have read up on so far... This would require being able to build a microsoft app store UWP (Universal Windows Platform) app no? This is a link to a microsoft tutorial on building a windows store app https://docs.microsoft.com/en-us/windows/uwp/get-started/your-first-app. I also found this article from Steve Dower http://stevedower.id.au/blog/why-so-many-python-installers/
. In it he mentions UWP in it and what some basic roadblocks are.
